src/schemes/fluid/weakly_compressible_sph/system.jl

```diff
@@ -358,14 +358,15 @@ function compute_surface_normal!(system, surface_tension::SurfaceTensionAkinci,
         v_neighbor_system = wrap_v(v_ode, neighbor_system, semi)
         nhs = get_neighborhood_search(system, semi)
 
-        calc_normal_akinci!(system, surface_tension, u, v_neighbor_system, u_neighbor_system, nhs,
+        calc_normal_akinci!(system, surface_tension, u, v_neighbor_system,
+                            u_neighbor_system, nhs,
                             neighbor_system)
     end
     return system
 end
 
-@inline function get_normal(particle, particle_system::FluidSystem,
-                            ::SurfaceTensionAkinci)
+@inline function surface_normal(particle, particle_system::FluidSystem,
+                                ::SurfaceTensionAkinci)
```

Comment:
```suggestion
@inline function surface_normal(::SurfaceTensionAkinci, particle_system::FluidSystem,
                                particle)
```
Consistency with other functions.